jsonschema-validation.xml

```diff
@@ -167,8 +162,8 @@
 
             <section title="Keywords and instance primitive types">
                 <t>
-                    Most validation keywords only limit the range of values within a certain primitive type.
-                    When the primitive type of the instance is not of the type targeted by the keyword, the
+                    Most validation keywords only exclude a range of values within a certain primitive type.
```

Comment:
What about

Most validation keywords only constrain values within a certain primitive type.

We've often talked about validation as a constraint system, and I think it feels more intuitive than talking about exclusion.